### DISCUSSION TITLE
Is there are way I can hide marks in the statuscolumn/signcolumn?

### DISCUSSION THREAD

**Discussion by mikker** [LazyVim/LazyVim#2644] [Q&A] [answered]
![CleanShot 2024-03-03 at 16 49 15@2x](https://github.com/LazyVim/LazyVim/assets/2819/fc212faf-06dc-47a9-8646-6f6eebd2e4d5)

See the `C` and `k` here. How do I make sign/status column (?) not show them? Is it possible? Not sure if this is a LazyVim issue specifically but I didn't have them before Lazy (I'm quite sure).

**Comment by dpetka2001** [comment, marked as answer]
Yes, this is specific to Lazyvim `statuscolumn`. You can either delete the marks with `:delmarks`, so they won't be shown, or you can set `vim.opt.statuscolumn = ""` in your `/lua/config/options.lua` to not show the LazyVim `statuscolumn` at all (that means you also won't have the other features that it provides) or just make your own custom `statuscolumn`.

**Reply by mikker** [reply]
Great, thank you!
